bin/triton-bootstrap

```diff
@@ -8,8 +8,14 @@
 #
 
 # This container's IP(s)
-export MYIPPRIVATE=$(ip addr show eth0 | grep -o '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}')
-export MYIPPUBLIC=$(ip addr show eth1 | grep -o '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}')
+export IP_PRIVATE=$(ip addr show eth0 | grep -o '[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}\.[0-9]\{1,3\}')
+IP_HAVEPUBLIC=$(ip link show | grep eth1)
+if [[ $IP_HAVEPUBLIC && ${IP_HAVEPUBLIC-x} ]]
```

Comment:
I've been trying to figure that out as well. It's a copy/paste that I'm glad you called out.